augur/merge.py

```diff
@@ -34,6 +34,13 @@
 future.  The SQLite 3 CLI, sqlite3, must be available.  If it's not on PATH (or
 you want to use a version different from what's on PATH), set the SQLITE3
 environment variable to path of the desired sqlite3 executable.
+
+Quotes are added around values with `certain characters`_ (array maps to `ASCII`_ positions).
+This should not impact downstream usage.
+
+.. _certain characters: https://sqlite.org/src/artifact/883d1470#:~:text=static%20const%20char%20needCsvQuote%5B%5D
+.. _ASCII: https://en.wikipedia.org/wiki/ASCII
```

Comment:
```suggestion
Quotes may be added around values that do not strictly need to be quoted, but
this should not impact downstream usage.
```

I'm really not sure the user-facing docs benefit from a link to the SQLite source code? It seems like it'll be noise for most users. And FWIW, it's not purely ASCII. ASCII is only single byte values 0-127. That array covers _all_ single byte values 0-255. The byte values in question are UTF-8 bytes, which overlap with ASCII for 0-127.

Reply:
Maybe just me then, I'll close this PR. We can reconsider if another user raises question about the quoting behavior.